deployment/ccip/changeset/state.go

```diff
@@ -166,6 +168,58 @@ type CCIPChainState struct {
 	RMN            *rmn_contract.RMNContract
 }
 
+func (c CCIPChainState) TokenAddressBySymbol() (map[TokenSymbol]common.Address, error) {
+	tokenAddresses := make(map[TokenSymbol]common.Address)
+	for symbol, token := range c.ERC20Tokens {
+		tokenAddresses[symbol] = token.Address()
+	}
+	for symbol, token := range c.ERC677Tokens {
+		tokenAddresses[symbol] = token.Address()
+	}
+	for symbol, token := range c.BurnMintTokens677 {
+		tokenAddresses[symbol] = token.Address()
+	}
```

Comment:
This seems like something useful to have as a helper on the `CCIPChainState` object, for two reasons (1) its used in this method and `TokenDetailsBySymbol` and (2) I wonder if we'd add some other token collection in the future?